dev-tools/packer/docker/xgo-image-deb6/beats-builder/gopacket_pcap.patch

```diff
@@ -7,20 +7,20 @@ index f5612e6..0c77efa 100644
 
  /*
 -#cgo linux LDFLAGS: -lpcap
-+#cgo linux,386 CFLAGS: -I /libpcap/i386/usr/include/
-+#cgo linux,386 LDFLAGS: /libpcap/i386/usr/lib/libpcap.a
-+#cgo linux,amd64 CFLAGS: -I /libpcap/amd64/libpcap-1.8.1
-+#cgo linux,amd64 LDFLAGS: /libpcap/amd64/libpcap-1.8.1/libpcap.a
++#cgo linux,386 CFLAGS: -I/libpcap/i386/usr/include/
++#cgo linux,386 LDFLAGS: -L/libpcap/i386/usr/lib/ -lpcap
++#cgo linux,amd64 CFLAGS: -I/libpcap/amd64/libpcap-1.8.1
++#cgo linux,amd64 LDFLAGS: -L/libpcap/amd64/libpcap-1.8.1 -lpcap
```

Comment:
Can you verify that Packetbeat is still statically linked with libpcap?

Reply:
@andrewkroh Already did on slack but should have added it to this issue.
```
vagrant@vagrant-ubuntu-precise-64:/vagrant/build/upload/packetbeat/packetbeat-7.0.0-alpha1-SNAPSHOT-linux-x86_64$ ldd packetbeat
        linux-vdso.so.1 =>  (0x00007ffe0c762000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fc835d19000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fc835b15000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fc835754000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fc835f36000)
```